class NotFoundException(Exception):
    pass


class CommonException(Exception):
    pass